crates/bevy_app/src/plugin.rs

```diff
@@ -162,7 +162,8 @@ mod sealed {
     }
 
     macro_rules! impl_plugins_tuples {
-        ($(($param: ident, $plugins: ident)),*) => {
+        ($(#[$meta:meta])* $(($param: ident, $plugins: ident)),*) => {
+            $(#[$meta])*
             impl<$($param, $plugins),*> Plugins<(PluginsTupleMarker, $($param,)*)> for ($($plugins,)*)
```

Comment:
This impl is for the internal `Plugins` trait, not the public one, so I think it doesn't show up in the docs.

Reply:
`cargo doc` supports --document-private-items, to document private items, which is a useful tool for engine developers.

Reply:
Ah, right. Never mind my comment then :)